README.md

```diff
@@ -144,6 +144,8 @@ of something that's broken.
 - Foreign elements
 - Rendering properties/hints
 
+## Adobe Illustrator SVG Configuration 
+![Export configuration](https://user-images.githubusercontent.com/2842459/62599914-91de9c00-b8fe-11e9-8fb7-4af57d5100f7.png)
```

Comment:
Maybe `Recommended Adobe Illustrator SVG Configuration`.

It would be nice to have an explanation of these settings and why they're recommended over the defaults

Reply:
Ok..I will update it

Reply:
@dnfield I updated it